pkg/operator/ceph/object/controller.go

```diff
@@ -456,9 +463,11 @@ func (r *ReconcileCephObjectStore) reconcileCreateObjectStore(cephObjectStore *c
 		if err != nil {
 			return reconcile.Result{}, errors.Wrapf(err, "failed to create object store %q", cephObjectStore.Name)
 		}
+
 	}
 
-	return reconcile.Result{}, nil
+	// Create COSI user and secret
+	return r.reconcileCOSIUser(cephObjectStore)
```

Comment:
I guess we should call this all when `BucketClass and BucketAccessClass` CRs are applied

Reply:
This is a special user just for COSI, we shouldn't need to create this for all OBCs or COSI buckets.

Reply:
Correct